<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_einstellung_wkz_pp</class>
 <widget class="QDialog" name="Dialog_einstellung_wkz_pp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Werkzeug Postprozessor</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>382</width>
     <height>107</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>folgende Werkzeugmagazine für die Bearbeitung von Werkstücken verwenden:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_wkz_fr_fmc">
      <property name="text">
       <string>Werkzeug fmc</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_wkz_fr_ganx">
      <property name="text">
       <string>Werkzeug ganx</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_wkz_fr_cix">
      <property name="text">
       <string>Werkzeug cix</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_wkz_fr_ggf">
      <property name="text">
       <string>Werkzeug ggf</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
